### PR TITLE
fix: Adjust memory and web worker count

### PR DESCRIPTION
**Pull request by pvenigal**
re: incident-213 

*Add a label `automerge` to automatically merge the PR after approval and passed checks.*

### CODE REVIEW

File: charts/datafold/values.yaml [hunk 1]
```diff
@@ -82,7 +82,7 @@ nginx:
   install: true
 
 config:
-  webWorkers: "5"
+  webWorkers: "4"
```
Comment (gtoonstra) [inline charts/datafold/values.yaml:85]: if replicacount is 2, this probably should be 2.

File: charts/datafold/charts/server/values.yaml [hunk 2]
```diff
@@ -41,10 +41,10 @@ ingress:
 
 resources:
   limits:
-    memory: 3500Mi
+    memory: 5500Mi
   requests:
     cpu: 500m
-    memory: 3500Mi
+    memory: 5500Mi
```
Comment (gtoonstra) [inline charts/datafold/charts/server/values.yaml:47]: I calibrated memory to make pods fit in 64G nodes. That means we need to leave a bit of headroom. With 2x5500 we go over that, so we always run with 2 nodes.

Comment (pvenigal) [reply charts/datafold/charts/server/values.yaml]: updated to 7000Mi and made replica count 1, keeping webWorkers at 4

File: charts/datafold/values.yaml (hunk 1 above)
Comment (pvenigal) [reply charts/datafold/values.yaml:85]: changed as per discussion to 1

**Review by gtoonstra — APPROVED** [review]

**Pull request merged**